sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloPropertiesTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.orchestration.center.instance;
+
+import org.junit.Test;
+import java.util.Properties;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class ApolloPropertiesTest {
+
+    @Test
+    public void assertGetValue() {
+        Properties props = new Properties();
+        props.setProperty(ApolloPropertyKey.APP_ID.getKey(), "APOLLO_SHARDING_SPHERE1");
+        props.setProperty(ApolloPropertyKey.ENV.getKey(), "PROD");
+        props.setProperty(ApolloPropertyKey.CLUSTER_NAME.getKey(), "sit");
+        props.setProperty(ApolloPropertyKey.ADMINISTRATOR.getKey(), "guest");
+        props.setProperty(ApolloPropertyKey.TOKEN.getKey(), "a6e6bbec-1eed-4ddc-bb10-6791beb982d9");
+        props.setProperty(ApolloPropertyKey.PORTAL_URL.getKey(), "http://apollo.ctrip.sit");
```

Comment:
"http://config-service-url"  should be better here.

Reply:
Thanks for your advice